### ISSUE TITLE
depans: support YUVA

### ISSUE THREAD

**Issue by realfinder** [pinterf/mvtools#43]
I try it with
```
AddAlphaPlane
mdata = DePanEstimate(trust=0)
DePan(data=mdata,offset=-1,matchfields=false,subpixel=2)
ExtractA()
```
![image](https://user-images.githubusercontent.com/2512153/120507846-d7bbfc80-c3cf-11eb-8593-e4d5ea226d06.png)
I need this fake YUVA to make DePan generate (zero?) alpha for black borders so I can make stab3() work faster in avs+ after some edits

**Comment by pinterf** [comment]
Seems easy and reasonable.

**Comment by realfinder** [comment]
maybe with setting special frame properties for new black borders or even mirror in this case (something like DePan_left, DePan_top, etc..), something like this https://github.com/HomeOfVapourSynthEvolution/VapourSynth-AddGrain/issues/4 but as writing

**Comment by pinterf** [comment]
Thinking it over, there can be a possible rotation and zoom as well (when MDepan originated vectors are used) so generating the new border is not that easy as for the case of perfectly horizontal/vertical shift. Did not look into the code yet though.

**Comment by realfinder** [comment]
yes, that why YUVA method seems better overall, but in simple case like stab() frame properties can work (stab() has zoom parameter but it's not used by default), rotation cases can't use frame properties method indeed, but for zoom I think it can
